xpk.py

```diff
@@ -1539,9 +1594,11 @@ def cluster_cacheimage(args) -> int:
   set_cluster_command_code = set_cluster_command(args)
   if set_cluster_command_code != 0:
     xpk_exit(set_cluster_command_code)
-
+  nodeSelectorKey = "cloud.google.com/gke-tpu-accelerator" if args.tpu_type else "cloud.google.com/gke-accelerator"
```

Comment:
We can use the AcceleratorCharertistics here to avoid separate hard coded names

Reply:
done